packages/cli/src/Server.ts

```diff
@@ -1085,12 +1087,15 @@ class App {
 
 					const sessionId = GenericHelpers.getSessionId(req);
 
+					const pinnedTrigger = findPinnedTrigger(workflowData, pinData);
```

Comment:
How do we handle the scenario in which we have multiple pinned triggers? WIll only one get executed?